src/calculateRentalCost.js

```diff
@@ -1,10 +1,22 @@
+'use strict';
 /**
  * @param {number} days
  *
  * @return {number}
  */
+
 function calculateRentalCost(days) {
-  // write code here
+  const rentalCost = days * 40;
+
+  if (days >= 7) {
+    return rentalCost - 50;
+  }
+
+  if (days >= 3) {
+    return rentalCost - 20;
```

Comment:
Similarly, use constants for the short-term discount threshold and the discount amount. For example, `const SHORT_TERM_RENTAL = 3;` and `const SHORT_TERM_DISCOUNT = 20;`.